src/containerapp/azext_containerapp/_params.py

```diff
@@ -276,6 +276,11 @@ def load_arguments(self, _):
         c.argument('user_assigned', nargs='+', help="Space-separated user identities to be assigned.")
         c.argument('system_assigned', help="Boolean indicating whether to assign system-assigned identity.", action='store_true')
 
+    with self.argument_context('containerapp up', arg_group='Deploy an Azure AI Foundry Model', is_preview=True) as c:
```

Comment:
Hi @harryli0108  @naziml 

Should we add an example in the `--help` for this feature to customer?

Reply:
Added example in latest commit